st2actions/st2actions/config.py

```diff
@@ -71,9 +71,9 @@ def _register_action_runner_opts():
         cfg.IntOpt('max_parallel_actions', default=50,
                    help='Max number of parallel remote SSH actions that should be run.  ' +
                         'Works only with Paramiko SSH runner.'),
-        cfg.BoolOpt('use_ssh_config',
-                    default=False,
-                    help='Use the .ssh/config file. Useful to override ports etc.')
+        cfg.StrOpt('ssh_config_path',
```

Comment:
Wait - so do we want two config options or not - `use_ssh_config` and `ssh_config_path`?

If we only have one (`ssh_config_path`), we should default it to `None` or similar for backward compatibility. Another option is to have two options, default `use_ssh_config` to `False` (same as before) and then we can leave default value for `ssh_config_path` as is.